ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -12,6 +10,48 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="nanmedian"),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
+)
+def test_numpy_nanmedian(
+    dtype_and_a,
+    dtype,
+    where,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    keep_dims,
+):
+    input_dtype, a, axis = dtype_and_a
+    if isinstance(axis, tuple):
+        axis = axis[0]
+    where, input_dtypes, test_flags = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtypes,
```

Comment:
F821 undefined name 'input_dtypes'

Reply:
done
what is your suggestion for this PR to be merged?
thanks